src/main/java/com/segment/analytics/android/integrations/mixpanel/MixpanelIntegration.java

```diff
@@ -125,8 +125,10 @@ public MixpanelIntegration(MixpanelAPI mixpanel, MixpanelAPI.People mixpanelPeop
   @Override public void identify(IdentifyPayload identify) {
     super.identify(identify);
     String userId = identify.userId();
-    mixpanel.identify(userId);
-    logger.verbose("mixpanel.identify(%s)", userId);
+    if (userId != null && userId.length() != 0) {
```

Comment:
I don't think we need the 2nd check. When is userId ever ""? IIRC our library doesn't even allow it.